### ISSUE TITLE
Decide what to do with discrete parameters in PINTS

### ISSUE THREAD

**Issue by ben18785** [pints-team/pints#682]
To begin with we should probably state that PINTS does not allow discrete parameters.

Later on, we should probably implement some discrete samplers. Not difficult to do so, but will require a bit a bit of fiddling. Should encourage users to marginalise out their discrete variables when possible.

**Comment by ben18785** [comment]
List of potential discrete samplers:

- [Continuous relaxation HMC](Additionally, any way we could facilitate marginalising out of discrete parameters would allow users to estimate their models in PINTS as they stand).

**Comment by MichaelClerx** [comment]
Just decided in meeting that current solution is:

- [ ] To begin with we should probably state that PINTS does not allow discrete parameters.

**Comment by ben18785** [comment]
Yep, in the README -- I can't think where it would go in the docs (other than in lots of places!)
